src/main/kotlin/link/infra/packwiz/installer/ui/gui/ExceptionListWindow.kt

```diff
@@ -24,6 +24,18 @@ class ExceptionListWindow(eList: List<ExceptionDetails>, future: CompletableFutu
 		fun getExceptionAt(index: Int) = details[index].exception
 	}
 
+	private fun openUrl(url: String) {
+		try {
+			if (Desktop.isDesktopSupported() && Desktop.getDesktop().isSupported(Desktop.Action.BROWSE)) {
+				Desktop.getDesktop().browse(URI(url))
+			} else {
+				Runtime.getRuntime().exec(arrayOf("xdg-open", url));
+			}
+		} catch (e: IOException) {
+			// lol the button just won't work i guess
```

Comment:
Don't let old sloppy comp500 code stick around... this should be at least a call to `Log`, maybe a JOptionPane error dialog.

Reply:
I'll give it a shot, but fair warning this is my first time writing Kotlin or any Java application for that matter haha

Reply:
Added the Log calls